src/Pages/Main/Components/GallerySlider.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useRef, useEffect } from 'react';
+import styled from 'styled-components';
+
+const GallerySlider = ({ children }) => {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef(null);
+
+  const goPrev = () => {
+    if (currentSlide === 0) {
+      setCurrentSlide(0);
+    } else {
+      setCurrentSlide(currentSlide - 1);
+    }
+  };
+
+  const goNext = () => {
+    if (currentSlide === slideRef.current.children[0].children.length) {
```

Comment:
- 복잡하게 ref 를 통해 접근하는 것 보다, 간단하게 children 을 이용하는 방법이 있습니다!
```suggestion
if (currentSlide === children.length) {}
```